### PR TITLE
update to support third-party cookie mitigation

### PR DESCRIPTION
**Pull request by alisaduncan**
- [x] App requests `offline_access` scope
- [x] Okta CLI config updated to include Refresh Token instructions
- [x] README.md instructions updated to walk the reader through manually enabling Refresh Token
- [x] Verified that the application runs and the user can sign in and sign out

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ axios
         clientID: CLIENT_ID,
         clientSecret: CLIENT_SECRET,
         callbackURL: 'http://localhost:3000/authorization-code/callback',
-        scope: 'groups profile offline_access',
```
Comment (edunham) [inline app.js:58]: I do not know why removing groups would be necessary to make 3pcd work

**Review by edunham — APPROVED** [review]
This looks to me like it does the steps which I understand are necessary for 3pcd. It also does a step that I do not know to be necessary, but looks probably harmless?

File: app.js (hunk 1 above)
Comment (alisaduncan) [reply app.js:58]: It's not necessary for the third-party cookies sample effort. The groups scope is not necessary for this app and is confusing.

I want to set a baseline for subsequent work, and I don't want groups' scope to be added everywhere. And will resolve #6 as a bonus.

Comment (edunham) [reply app.js:58]: perfect, big +1 on fixing other bugs whenever it's easy. I assumed it was something like this =)

**Pull request merged**